barman/config.py

```diff
@@ -1930,12 +1930,77 @@ def receive_config_changes(self, changes):
         # Get all the available configuration change files in order
         changes_list = []
         for section in changes:
-            section_name = section["server_name"]
+            original_section = deepcopy(section)
+            section_name = None
+            scope = section.pop("scope")
+
+            if scope not in ["server", "model"]:
```

Comment:
I think we want to move this validation section in a separate method? not now but in the future... for cleaner code. maybe is just me overthinking btw.

Reply:
I think that makes sense, yes.
I guess we could do that altogether later, when implementing the more advanced validations to the command?